backend/src/main/java/com/github/EazyErik/Service/DrinksService.java

```diff
@@ -0,0 +1,11 @@
+package com.github.EazyErik.Service;
+
+import com.github.EazyErik.Repository.DrinksRepository;
+import org.springframework.stereotype.Service;
+
+@Service
+public class DrinksService {
+
+    public DrinksRepository drinksRepository;
```

Comment:
Fields should be private. Also this should be final. That way you make sure, that it won't be overwritten and can use `@RequiredArgsConstructor`.